docker/Dockerfile

```diff
@@ -59,14 +59,20 @@ RUN \
   && apt-get install -y --no-install-recommends nodejs npm \
   && npm install -g @datadog/datadog-ci \
   # Install Mariadb last because mariadb_repo_setup currently breaks apt
-  && wget https://downloads.mariadb.com/MariaDB/mariadb_repo_setup \
-  && chmod +x mariadb_repo_setup \
-  && ./mariadb_repo_setup \
+  && wget -O- https://mariadb.org/mariadb_release_signing_key.pgp |gpg --dearmor > /etc/apt/trusted.gpg.d/mariadb.gpg \
+  && echo 'deb [arch=amd64,arm64] https://mirror.mariadb.org/repo/11.rolling/debian/ buster main' > /etc/apt/sources.list.d/mariadb.list \
```

Comment:
You should need an `apt-get update` after adding a new repo. Maybe this is installing from the Debian repos and not from the upstream one?

Reply:
Ah, yes, that was probably taken care of by `mariadb_repo_setup` before, but regardless, we're going to move those changes to #11609 anyway.